sw/device/silicon_creator/manuf/lib/personalize.c

```diff
@@ -308,8 +190,14 @@ static status_t otp_partition_secret2_configure(
 
 status_t manuf_personalize_device_secrets(
     dif_flash_ctrl_state_t *flash_state, const dif_lc_ctrl_t *lc_ctrl,
-    const dif_otp_ctrl_t *otp_ctrl, ecc_p256_public_key_t *host_ecc_pk,
-    wrapped_rma_unlock_token_t *wrapped_rma_token) {
+    const dif_otp_ctrl_t *otp_ctrl,
+    const lc_token_hash_t *rma_unlock_token_hash) {
+  ;
+  LOG_INFO("Provisioning OTP SECRET2 and keymgr flash info pages ...");
+
+  // Perform OTP and flash info writes.
+  LOG_INFO("Provisioning OTP SECRET2 and keymgr flash info pages ...");
```

Comment:
```suggestion
```
We should try to keep console prints in the test itself, not in the libs, since we will want to optimize/remove console prints later on to optimize provisioning throughput. (Also, this looks accidentally duplicated.)

Reply:
oops, not sure how I missed this, too many back and forths